common4j/src/main/com/microsoft/identity/common/java/crypto/IDevicePopManager.java

```diff
@@ -95,11 +88,11 @@
      * requires use of a SHA-1 digest or uses NO_PADDING should not be supported.
      */
     enum Cipher implements AsymmetricAlgorithm {
-        @RequiresApi(Build.VERSION_CODES.JELLY_BEAN_MR2)
+        //@RequiresApi(Build.VERSION_CODES.JELLY_BEAN_MR2)
```

Comment:
Not needed? Should the line be just removed?

Reply:
I'm keeping this for reference. It's still useful.. even if it won't show up warning.

(RequiresApi cannot be used in java project)